resources/templates/node/Dockerfile.template

```diff
@@ -1,5 +1,8 @@
 FROM node:14-alpine
 ENV NODE_ENV=production
+{{#unless (isRootPort ports)}}

```

Comment:
Incidentally I found that the Node image let me use ports < 1024 while not being root, so if we want we can probably remove this guard.

Reply:
I think I'd remove that just from a consistency point of view (i.e. we scaffold using `node` user but then user changes the port being used or vice-versa).